com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -108,14 +108,14 @@
     public static final String ERROR_DETERMINING_ACTIONS_TO_EXECUTE_ON_APP = "Error determining actions to execute on application \"{0}\"";
     public static final String ERROR_STOPPING_APP = "Error stopping application \"{0}\"";
     public static final String ERROR_STAGING_APP_1 = "Error staging application \"{0}\"";
-    public static final String ERROR_STAGING_APP_2 = "Error staging application \"{0}\": {1}";
+    public static final String ERROR_STAGING_APP_2 = "Staging of application \"{0}\" has failed. Check the application logs downloaded from the dmol command";

```

Comment:
I'd change it to `"Staging of application \"{0}\" failed. Download the application logs via the dmol command and check them for more information."`.